app/controllers/users_controller.rb

```diff
@@ -11,17 +18,43 @@ def create
     @user = User.new(user_params)
     if @user.save
       flash[:success] = "Saved successfully!"   
-      redirect_to @user
+      redirect_to login_url
     else
       flash[:danger] = "Invalid content. Try again."
       render 'new'
     end
   end
 
+  def edit
+    @user = User.find(params[:id])
+  end
+
+  def update
+    @user = User.find(params[:id])
+    if @user.update_attributes(user_params)
+      flash[:success] = "Saved successfully!"
+      redirect_to @user
+    else
+      dlash[:danger] = "Invalid content. Try again."
```

Comment:
![Screen Shot 2021-05-17 at 9 22 15 PM](https://user-images.githubusercontent.com/35232348/118495684-02625000-b756-11eb-8fd8-972a211cb81b.png)
You have a misspelling right here please fix.